beavy/models/object.py

```diff
@@ -14,7 +14,7 @@
 
 class ObjectQuery(AccessQuery):
 
-    def by_capability(self, aborting=True, abort_code=404, *caps):
+    def by_capability(self, *caps, aborting=True, abort_code=404):
```

Comment:
This was causing an error because of the order of the args list. In the commit, I wrote:

> Was getting error related to order of arguments: "got multiple
values for argument...", in this case the two keyword args. This
seems to fix it. See https://stackoverflow.com/questions/28336270/how-to-fix-got-multiple-values-for-argument-error-for-args-and-kwargs